src/main/webapp/app/exam/participate/summary/result-overview/exam-result-overview.component.scss

```diff
@@ -0,0 +1,58 @@
+#result-overview-table tr:not(thead tr):not(tfoot tr) {
+    cursor: pointer;
+
+    &:hover {
+        transform: scale(1.01);
+        transition: transform 0.1s;
+    }
+}
+
+.not-included-in-score {
+    opacity: 0.5;
+}
+
+.grade {
+    color: var(--white);
+    padding: 0.25em 0.5em;
+    margin-left: 0.25em;
+    border-radius: 0.25em;
+}
+
+.passing-grade {
+    background-color: var(--success);
+}
+
+.failing-grade {
+    background-color: var(--danger);
+}
+
+.total-footer {
+    border-top-width: 2px;
+}
+
+.chevron-position {
+    display: inline-block;
+    vertical-align: middle;
+}
+
+.rotate-icon {
+    transition: transform 0.3s ease;
+}
+
+.rotate-icon.rotated {
+    transform: rotate(90deg);
+}
+
+.icon-container {
+    display: inline-block;
+    vertical-align: middle;
+    horiz-align: center;
```

Comment:
I don't think that exists? Maybe look it up?

Reply:
Addressed in https://github.com/ls1intum/Artemis/pull/7281/commits/24bd8bf73203f2a557c885dc7956ee78a5e8f7f1